bin/docker_start

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require 'uri'
+
+# check that postgres is up
```

Comment:
Do we need to wait for the database user to exist like we do in our various circle.yml files?

Reply:
added